diesel_cli/src/main.rs

```diff
@@ -1,5 +1,20 @@
+// Built-in Lints
+#![deny(warnings, missing_copy_implementations)]
+
+// Clippy lints
+#![cfg_attr(feature = "clippy", allow(unstable_features))]
```

Comment:
Actually, if we put that in the root directory, how can we make sure the `include!`d file gets, well, included in the `cargo package` artifact?